src/api/custom-resolvers/mutations/create-an-account/index.test.ts

```diff
@@ -58,71 +63,164 @@ describe('custom-resolvers/create-an-account', () => {
     confirmEmailAddressEmail.send = sendConfirmEmailAddressEmailSpy;
 
     await accounts.deleteAll(context);
-
-    // create an account
-    account = (await createAnAccount({}, variables, context)) as Account;
-
-    /**
-     * Get the fully created account.
-     * note: the response only returns some specific fields.
-     */
-    createdAccount = await accounts.get(context, account.id);
-
-    return createdAccount;
   };
 
   describe('when an account with the provided email does NOT already exist', () => {
+    beforeAll(() => {
+      process.env.NODE_ENV = 'not-development';
+    });
+
     beforeEach(async () => {
-      createdAccount = await setupTests();
+      jest.resetAllMocks();
+
+      await setupTests();
+
+      createdAccount = await createAnAccount({}, variables, context);
     });
 
-    it('should generate a created account with added salt and hashes', () => {
-      expect(createdAccount.firstName).toEqual(variables.firstName);
-      expect(createdAccount.lastName).toEqual(variables.lastName);
-      expect(createdAccount.email).toEqual(variables.email);
-      expect(createdAccount.salt.length).toEqual(RANDOM_BYTES_SIZE * 2);
-      expect(createdAccount.hash.length).toEqual(KEY_LENGTH * 2);
-      expect(createdAccount.verificationHash.length).toEqual(KEY_LENGTH * 2);
+    // it('should return the account ID, verification hash and success=true', async () => {
```

Comment:
should this be removed? or TODO comment?

Reply:
Sorry, PR didn't have the latest commit.